hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMMXBean.java

```diff
@@ -80,4 +80,7 @@ public interface SCMMXBean extends ServiceRuntimeInfo {
    */
   String getPrimordialNode();
 
+  String getScmRatisLogDirectory();
+
+  String getScmRocksDbDirectory();
```

Comment:
suggest to drop the word "Scm" because everything in this MXBean is for SCM. It's redundant.
```suggestion
  String getRatisLogDirectory();

  String getRocksDbDirectory();
```

Reply:
Thanks for the suggestion !!